libertai_agents/libertai_agents/agents.py

```diff
@@ -155,6 +160,9 @@ async def __api_generate_answer(
         Generate an answer based on an existing conversation.
         The response messages can be streamed or sent in a single block.
         """
+        if self.call_session is None:
+            self.call_session = ClientSession()
```

Comment:
not sure that we should always use the same session for IP calls, because it might be an agent used in a UI by several users, not a single one.
For Python calls (Telegram bot for example), the developer can store a dict of sessions and pass the right one each time, but via API he can't (or we have to find a way)